### DISCUSSION TITLE
续费了2年的专业版，但是无法通过邮箱登陆，无法使用专业版，无法收到验证码，咋解决，要么退款，要么解决问题

### DISCUSSION THREAD

**Discussion by whpublic2024** [getlantern/lantern-forum-cn#758] [Q&A]
邮箱为：1013009448@qq.com

支付凭证如下：
![Uploading WechatIMG354.jpg…]()

**Comment by whpublic2024** [comment]
谁能解决问题？

**Comment by whpublic2024** [comment]
图片还上传不了

**Reply by MaxMKMax** [reply]
@whpublic2024 您好. 对的, 我这里看不到您的图片. 你发到我的邮箱吧. 我帮您解决问题. 

mike@getlantern.org

收不到激活码是因为国内限制. 所以邮箱过不去. 我这里会帮您拿激活码.

**Reply by whpublic2024** [reply]
给你发邮件了。　辛苦快点解决。

**Reply by clarkpkchyg** [reply]
> @whpublic2024 您好. 对的, 我这里看不到您的图片. 你发到我的邮箱吧. 我帮您解决问题.
> 
> [mike@getlantern.org](mailto:mike@getlantern.org)
> 
> 收不到激活码是因为国内限制. 所以邮箱过不去. 我这里会帮您拿激活码

我也拿不到激活码，已经发邮件给您了，clarkpkchyg@sina.com

**Comment by whpublic2024** [comment]
已经发邮件了。

**Comment by whpublic2024** [comment]
也给　mailto:mike@getlantern.org　发送邮件了，为啥还是没处理呢？？？

**Comment by whpublic2024** [comment]
为啥还不处理呀

**Comment by whpublic2024** [comment]
激活码怎么不给发啊

**Comment by whpublic2024** [comment]
激活码怎么不给发啊

**Comment by whpublic2024** [comment]
邮件已经发了好几天了，怎么还是不处理？

**Comment by whpublic2024** [comment]
邮件已经发了好几天了，怎么还是不处理？
邮箱为：1013009448@qq.com

**Comment by whpublic2024** [comment]
@Mike777Khoo

**Comment by whpublic2024** [comment]
@Mike777Khoo

**Comment by whpublic2024** [comment]
@Mike777Khoo 
邮件已经发了好几天了，怎么还是不处理？
邮箱为：1013009448@qq.com

**Comment by whpublic2024** [comment]
好几年的老用户了

**Comment by whpublic2024** [comment]
非常失望啊。　一直不解决问题。

**Comment by whpublic2024** [comment]
再不解决，就在 alipay 上投诉商户交易诈骗了。

**Comment by whpublic2024** [comment]
蓝灯怎么了，不解决问题？

**Comment by clarkpkchyg** [comment]
我已经找客服了，他们说帮我要一下商家的联系方式

whpublic2024 ***@***.***> 于2024年8月9日周五 10:33写道：

> 再不解决，就在 alipay 上投诉商户交易诈骗了。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/getlantern/lantern-forum-cn/discussions/758#discussioncomment-10281463>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKM5JFHDWLBJNNS2UVZNCJ3ZQQTABAVCNFSM6AAAAABMATURXGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRYGE2DMMY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> .com>
>


-- 
陈亚光
首都经济贸易大学外语学院 本科三年级
***@***.***
联系方式：86-13681252758
Yaguang Chen
Capital University of Business and Economics
Foreign Language Department
BA in Business English
Secretary-General, CUEB Model United Nations Association
Cell:86-13681252758
***@***.***

**Reply by whpublic2024** [reply]
你的也是没解决吗?

**Comment by whpublic2024** [comment]
怎么还不解决问题？

**Comment by whpublic2024** [comment]
感觉这个品牌开始摆烂了，没有诚信。

**Comment by whpublic2024** [comment]
@Mike777Khoo   你让我发邮件，为什么发了这么久，仍然不处理呢？

**Comment by clarkpkchyg** [comment]
我是通过支付宝付款的，我联系了支付宝客服，他们说会提供商家的联系方式，说自己和商家沟通

whpublic2024 ***@***.***> 于2024年8月13日周二 10:03写道：

> @Mike777Khoo <https://github.com/Mike777Khoo> 你让我发邮件，为什么发了这么久，仍然不处理呢？
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/getlantern/lantern-forum-cn/discussions/758#discussioncomment-10320238>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKM5JFCSLBHCRDL5UTYUTLLZRFSQFAVCNFSM6AAAAABMATURXGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZSGAZDGOA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> .com>
>


-- 
陈亚光
首都经济贸易大学外语学院 本科三年级
***@***.***
联系方式：86-13681252758
Yaguang Chen
Capital University of Business and Economics
Foreign Language Department
BA in Business English
Secretary-General, CUEB Model United Nations Association
Cell:86-13681252758
***@***.***

**Reply by whpublic2024** [reply]
他们联系不上啊

**Comment by whpublic2024** [comment]
解决问题不，品牌信誉也不要了吗

**Comment by whpublic2024** [comment]
为啥迟迟不解决问题？

**Comment by whpublic2024** [comment]
已经 2 周多了，还没有解决问题。

**Comment by whpublic2024** [comment]
啥时候解决问题啊?

**Reply by MaxMKMax** [reply]
@whpublic2024 您好. 不好意思漏掉您的帖子了. 我给您申请 1-2 个月的补偿. 
超级抱歉的. 已经给您把问题提交上去了. 谢谢您支持蓝灯. 真的非常抱歉!

**Comment by MaxMKMax** [comment]
@whpublic2024 
您好. 这是您的验证码:
JR4VV-8QJQH-4V237-7R9GH-K9F4Y
您登陆好了之后通知我. 我让技术人员给您加时间补偿您.
